### PR TITLE
Add getGroupKeyInfo method

### PR DESCRIPTION
**Pull request by Powersource**

### CODE REVIEW

**Comment by Powersource** [comment]
@staltz once approved you can merge and publish a new version :)

File: format.js [hunk 1]
```diff
@@ -77,6 +77,16 @@ function makeEncryptionFormat() {
     })
   }
 
+  function getGroupKeyInfo(groupId, cb) {
+    if (cb === undefined) return promisify(getGroupKeyInfo)(groupId)
+
+    if (!groupId) cb('Group id required')
```
Comment (staltz) [inline format.js:83]:
```suggestion
    if (!groupId) cb(new Error('Group id required'))
```

**Review by staltz — CHANGES_REQUESTED** [review]
@Powersource it would be good if PRs would have at least some description of *why* the change is needed. This information is not just for the reviewer, it's also for the future when we (or others) trace when was a change made, and for what reasons.

At the moment I'm not quite sure exactly why you need this method on ssb-box2 (working on a lot of modules and problems at once, I lose context easily).

**Comment by Powersource** [comment]
@staltz good point. I'm trying to implement tribes2.list() and for that i need to be able to get a group's secret/key (among other things, not sure how I'm getting those yet) 

https://github.com/ssbc/ssb-tribes2/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R65-R68

**Comment by Powersource** [comment]
Is it important for list() to return the group's key?

**Pull request merged**